frontend/src/components/Following/FollowingButton.js

```diff
@@ -1,69 +1,74 @@
 import '../../config';
 import axios from "axios";
+import LocalStorageService from '../../services/LocalStorageService';
 import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
 
 import './FollowingButton.scss'
 
 export default function FollowingButton(props) {
-    const [isFollowing, setIsFollowing] = useState(false);
-    const [errorMsg, setErrorMsg] = useState("");
+    const [isFollowing, setIsFollowing] = useState();
+    const history = useHistory();
+    let errorMsg = "";
```

Comment:
Why was this changed? It's better practice to have constants rather than `let`s in your code

Reply:
I changed from a state variable to a normal variable because the error msg doesn't act like a state. It's a param that goes into an alert. So it makes more sense to have it as a variable. As for why I didn't put const, it's because the value of errorMsg has to be modified later. From an empty string to the error message. If it was a const, it would be immutable.

Reply:
Yeah, I understand. For some reason I can't quite recall, I remember always being asked to use the React Hook setter/getters that way, even if they weren't explicitly state variables...because the `const [var, settter]` format only manages local state. I know I find it clearner and more intuitive to answer, but I can't remember an exact rule stating that`const` is definitely prioritized over `let` as being better practice but I know it was perceived as so in my old workplace. But yeah, since I can't recall any definite rule I guess we can keep it this way